resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -0,0 +1,415 @@
+---
+
+- slug: trimmomatic-single
+  name: trimmomatic (single-end)
+  data_name: '{{ reads.fastq.0.file|default:"?" }}'
+  version: 0.0.1
+  type: data:reads:fastq:single:trimmomatic
+  category: analyses
+  flow_collection: sample
```

Comment:
All filtered reads are derived from the "data:reads:fastq" objects which itself are already part of the sample - filtered reads objects should thus be added to the already existing sample object. Does "flow_collection: sample" needs to be added in such cases?